src/sentry/static/sentry/app/views/dashboardsV2/widget/metricWidget/pickProjectToContinue.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import * as ReactRouter from 'react-router';
+import styled from '@emotion/styled';
+
+import {openModal} from 'app/actionCreators/modal';
+import ContextPickerModal from 'app/components/contextPickerModal';
+import {Organization, Project} from 'app/types';
+
+type Props = {
+  orgSlug: Organization['slug'];
+  router: ReactRouter.InjectedRouter;
+  projects: Project[];
+};
+
+const PickProjectToContinue = ({orgSlug, router, projects}: Props) => {
```

Comment:
This seems to be a duplication of the one in release detail - should we consolidate?

Reply:
sure... I am working a consolidation in this PR https://github.com/getsentry/sentry/pull/25183